### PR TITLE
[Job Clean] Sort out the cleanup logic of historical jobs

### PR DESCRIPTION
**Pull request by morningman**
## Proposed changes

There are many historical job records in Doris, such as load jobs, alter jobs, export jobs and so on.

These historical jobs are generally cleaned up periodically by the cleanup thread, to avoid taking too much memory.

This PR reorganized the cleanup logic of historical jobs and optimized the cleanup logic of some historical jobs
to reduce the memory usage of historical jobs.

The following FE configuration items are related to historical job cleaning:

1. label_keep_max_second

    Used to determine whether LoadJob, LoadJobV2, RoutineLoadJob or TransactionState are expired.

2. streaming_label_keep_max_second

    Used to determine whether InsertJob, DeleteJob  or TransactionState are expired.
    Different from label_keep_max_second, this config is used to clean up these frequently submitted jobs or load transactions.

3. history_job_keep_max_second

    Used to determine whether AlterJob, ExportJob are expired

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [x] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -101,80 +101,83 @@
      *          60m     60 mins
      *          120s    120 seconds
      */
-    @ConfField public static String audit_log_dir = PaloFe.DORIS_HOME_DIR + "/log";
-    @ConfField public static int audit_log_roll_num = 90;
-    @ConfField public static String[] audit_log_modules = {"slow_query", "query", "load", "stream_load"};
-    @ConfField(mutable = true) public static long qe_slow_log_ms = 5000;
-    @ConfField public static String audit_log_roll_interval = "DAY";
-    @ConfField public static String audit_log_delete_age = "30d";
+    @ConfField
```
Comment (xy720) [inline fe/fe-core/src/main/java/org/apache/doris/common/Config.java:104]: Will this code style be more standardized? Do we need to write in this style in the future?

**Review by EmmyMiao87 — DISMISSED** [review]
LGTM

**Review by xy720 — COMMENTED** [review]
This commit filters the expired ExportJob and DeleteHandler while saving image, why not do filter the load jobs for Load and LoadManager too?

**Comment by morningman** [comment]
> This commit filters the expired ExportJob and DeleteHandler while saving image, why not do filter the load jobs for Load and LoadManager too?

We already did this. See:

https://github.com/apache/incubator-doris/blob/56b2a33a4d07cf36ad16a63bd243b758b2111b22/fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadManager.java#L768-L770

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java (hunk 1 above)
Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/common/Config.java:104]: This is done by IDEA automatically, maybe its good to format like this?

**Review by xy720 — APPROVED** [review]
LGTM

**Review by EmmyMiao87 — APPROVED** [review]
LGTM

**Pull request merged**